2025/build-tooling/demo/final/vite.config.ts

```diff
@@ -1,10 +1,32 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
+import chaosMonkey from "./support/chaosMonkey.mjs";
+
+const API_URL = "/arcgis/rest/services/places-service";
```

Comment:
Add a comment explaining why proxy is needed?

Reply:
Moved this into a config file, but generally two reasons:
* API Key has referrer restrictions, so it goes through a proxy for local development to skip those
* To inject the chaos